stream-chat-android-offline/src/main/java/io/getstream/chat/android/livedata/controller/ChannelControllerImpl.kt

```diff
@@ -190,9 +186,6 @@ internal class ChannelControllerImpl(
     val channelController = client.channel(channelType, channelId)
     override val cid = "%s:%s".format(channelType, channelId)
 
-    private val job = SupervisorJob()
-    val scope = CoroutineScope(Dispatchers.IO + domainImpl.job + job)
```

Comment:
- This `job` was never cancelled, making it useless - the point of `Job` instances as a `CoroutineContext` and of `CoroutineScope` is cancellation.
- The `scope` created here is not a sub-scope of `domainImpl.job` in any way. Since `job` is also a `Job` element, it overwrites `domainImpl.job` in this combined `CoroutineContext`.